src/st/st0/st0.h

```diff
@@ -4,6 +4,8 @@
 #undef STAGE
 #define STAGE STAGE_ST0
 
+#define STAGE_IS_ST0
+
```

Comment:
Not for this PR as I do not want to stall it further. But I think this can now be removed?